src/plugins/mini/Examples/read.ini

```diff
@@ -0,0 +1,15 @@
+keyWithoutLeadingWhitespace = 	valueWithLeadingWhiteSpace
```

Comment:
> The installation of the files won't work then.

Oh, I did not know that.

> Please also add `INSTALL_TEST_DATA` so that your test can be executed after Elektra was installed.

Okay, I added the attribute in commit 6ca5e40d.